src/Umbraco.Web.UI.Client/src/common/services/minieditorhelper.service.js

```diff
@@ -0,0 +1,79 @@
+(function () {
+    'use strict';
+
+    function miniEditorHelper(dialogService, editorState, fileManager, contentEditingHelper) {
+
+        var launched = false;
+
+        function launchMiniEditor(node) {
```

Comment:
I don't think we should put this in our public api right now. It is a major hack and something we should find a better solution for as soon as possible (probably v. 8 a split views)